samples/unit-tests/accessibility/accessible-highmaps/demo.js

```diff
@@ -1,13 +1,16 @@
 QUnit.test('Basic map', function (assert) {
-    var chart = Highcharts.mapChart('container', {
+    const polishFlagURL =
+        'https://upload.wikimedia.org/wikipedia/en/1/12/Flag_of_Poland.svg';
+    const chart = Highcharts.mapChart('container', {
             chart: {
                 map: 'custom/europe'
             },
             series: [
                 {
+                    keys: ['hc-key', 'color.pattern.image'],
```

Comment:
You have 2 keys defined, but your data point has 3 values: `['pl', polishFlagURL, 2],`